Browser/FAHSetupDialog.vb

```diff
@@ -620,7 +620,7 @@
             End If
 
         Catch ex As Exception
-            Dim strMsg As String = "Telnet Error: " & ex.Message.ToString
+            Dim strMsg As String = "Telnet error while connecting to the Folding@Home software:" & vbNewLine & ex.Message.ToString & vbNewLine & vbNewLine & "-Please verify the Folding@Home software is running, and then try again."
```

Comment:
The last line of the error message there is a '-' at the beginning, is that meant to be there?

Reply:
Yes. It was intentional. So it would standout more being after the normal error message, like a list of things to try. There are other messages with listed items similar to this. Thanks for checking though.